### PR TITLE
Hand-encode ping request body to stay within 64 byte google.com limit

### LINKED ISSUE #143: Unable to decode ping response from google.com
When using `ping --host google.com`, it produces:

```
WARN  Failed to decode echo reply payload data=[50 255 129 3 1 1 11 101 99 104 111 80 97 121 108 111 97 100 1 255 130 0 1 2 1 10 73 100 101 110 116 105 102 105 101 114 1 12 0 1 4 83 101 110 116 1 255 132 0 0 0 16 255 131 5 1 1 4 84 105 109 101 1 255]
```

Which, converted to ASCII, does seems to show indications of our gob encoding:

```
2 ÿ      e c h o P a y l o a d  ÿ      
 I d e n t i f i e r      S e n t  ÿ      ÿ      T i m e  ÿ
```

### PR DESCRIPTION
**Pull request by itzg**
Fixes #143

### CODE REVIEW

File: check/check_ping.go [hunk 1]
```diff
@@ -41,7 +41,10 @@ func NewPingCheck(base *Base) Check {
 	check := &PingCheck{Base: *base}
 	err := json.Unmarshal(*base.RawDetails, &check.Details)
 	if err != nil {
-		log.Printf("Error unmarshalling check details")
+		log.WithFields(log.Fields{
+			"err":  err,
```
Comment (rphillips) [inline check/check_ping.go]: this one is missing a prefix.

Comment (itzg) [inline check/check_ping.go]: Added prefixes to this and the `pinger.go`

Comment (itzg) [inline check/check_ping.go]: oh, wait, that one is pre-decoding so I don't think it can generate a prefix

Comment (itzg) [inline check/check_ping.go]: ...for example, NewTCPCheck doesn't use a prefix in its log

Comment (rphillips) [inline check/check_ping.go]: ah ok

**Pull request merged**